server/monitor/src/main/java/org/apache/accumulo/monitor/util/ParameterValidator.java

```diff
@@ -22,6 +22,8 @@
 public interface ParameterValidator {
 
   String ALPHA_NUM_REGEX = "\\w+";
+  // Allow the special default table IDs
+  String ALPHA_NUM_REGEX_TABLE_ID = "[!+]?\\w+";
```

Comment:
It occurs to me that these all should probably be `final`. I think `public static` is implied since this is an interface, but I don't think `final` is implied. I'm actually surprised this works, because I thought you could only reference final strings in annotations. So, maybe `final` is implied for strings in interfaces?

Reply:
IntelliJ gives me the suggestion: "Modifier 'final' is redundant for interface fields"

Reply:
TIL. Awesome. :smile: